ERROR - 2022-11-09 11:23:55 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mysql'@'localhost' (using password: YES) in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'mysql', 'mysql', 'actipress', 3306, '', 0)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-09 11:23:55 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `utilisateurs`
WHERE `Pseudo` = :Pseudo:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(622): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Login->signIn()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-09 11:25:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mysql'@'localhost' (using password: YES) in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'mysql', 'mysql', 'actipress', 3306, '', 0)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-09 11:25:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `utilisateurs`
WHERE `Pseudo` = :Pseudo:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(622): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Login->signIn()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-09 11:25:39 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mysql'@'localhost' (using password: YES) in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'mysql', 'mysql', 'actipress', 3306, '', 0)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-09 11:25:39 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `utilisateurs`
WHERE `Pseudo` = :Pseudo:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(622): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Login->signIn()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-09 11:54:31 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 219.
 1 SYSTEMPATH\CodeIgniter.php(219): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(177): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
ERROR - 2022-11-09 14:51:29 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mysql'@'localhost' (using password: YES) in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'mysql', 'Nouga_123', 'actipress', 3306, '', 0)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-09 14:51:29 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `utilisateurs`
WHERE `Pseudo` = :Pseudo:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(622): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Login->signIn()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-09 14:52:13 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mysql'@'localhost' to database 'actipress' in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'mysql', 'mysql', 'actipress', 3306, '', 0)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' to database 'actipress' in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-09 14:52:13 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' to database 'actipress'
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `utilisateurs`
WHERE `Pseudo` = :Pseudo:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(622): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Login->signIn()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-09 14:52:32 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mysql'@'localhost' to database 'actipress' in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'mysql', 'mysql', 'actipress', 3306, '', 0)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' to database 'actipress' in C:\xampp\htdocs\actipress\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actipress\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actipress\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actipress\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actipress\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\actipress\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\actipress\system\CodeIgniter.php(907): App\Controllers\Login->signIn()
#7 C:\xampp\htdocs\actipress\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\actipress\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actipress\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-09 14:52:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' to database 'actipress'
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `utilisateurs`
WHERE `Pseudo` = :Pseudo:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(622): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Login->signIn()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
